### PR TITLE
courses controller created and model migrated

### PR DESCRIPTION
**Pull request by mlworle**
Just doing a smaller PR

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,14 @@
   Rails.application.routes.draw do
+  get 'courses/new'
```
Comment (edwardloveall) [inline config/routes.rb]: Are all these individual routes what you want? Looks like a great place to use `resources`

**Comment by mlworle** [comment]
Courses action and views up to same level as teachers and students.

File: app/controllers/courses_controller.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class CoursesController < ApplicationController
+  def new
```
Comment (edwardloveall) [inline app/controllers/courses_controller.rb]: Are these all supposed to be blank?

**Comment by edwardloveall** [comment]
Just gook a look and the views that you added seem fine, but there are no controller actions to back any of that up! Are those forthcoming or did you have a different plan in mind?

**Comment by mlworle** [comment]
Sorry! Had unstaged files.

File: app/views/courses/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<%= form_for(@course) do |f| %>
+  <% if @course.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@course.errors.count, "error") %> prohibited this student g from being saved:</h2>
```
Comment (edwardloveall) [inline app/views/courses/_form.html.erb]: student g?

**Comment by edwardloveall** [comment]
Yup. Looks good. :shipit:

**Pull request merged**